pkg/apis/ocmagent/v1alpha1/managednotification_types_test.go

```diff
@@ -1,287 +0,0 @@
-package v1alpha1_test
```

Comment:
Seems like we lost these tests.

Reply:
I have added in my latest commit. Thanks Matt